Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -63,7 +63,8 @@ @interface WMFExploreViewController () <WMFExploreSectionSchemaDelegate,
     UIViewControllerPreviewingDelegate,
     WMFAnalyticsContextProviding,
     WMFAnalyticsViewNameProviding,
-    WMFColumnarCollectionViewLayoutDelegate>
+    WMFColumnarCollectionViewLayoutDelegate,
+    UINavigationControllerDelegate>
```

Comment:
@joewalsh  what navigation controller is this the delegate of? I removed it the other day because I thought it had some extra delegate methods that didn't belong to it - did I mis something else?